fireant/slicer/filters.py

```diff
@@ -36,6 +42,11 @@ def __init__(self, element_key, values):
     def schemas(self, element):
         return element.isin(self.values)
 
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) \
+               and self.element_key == other.element_key \
+               and self.values == other.values
```

Comment:
I added this for some antfarm tests where I wanted to check if the equality filter used was the one expected.  In this case, I don't have a reference to the same instance so I need to check the equality.